src/presence_analyzer/templates/presence_top_5.html

```diff
@@ -0,0 +1,61 @@
+<%inherit file="base.html"/>
+
+<%block name="title">
+    <title>TOP 5</title>
+</%block>
+
+<%block name="head">
+    <script type="text/javascript">
+        (function($) {
+            $(document).ready(function() {
+                var $loading = $('#loading'),
+                    xhr = null;
+                $.getJSON("/api/v1/months", function(result) {
+                    var dropdown = $("#object_id");
+                    $.each(result, function(item) {
+                        dropdown.append($("<option />").val(result[item]).text(result[item]));
```

Comment:
ditto
and this imo is more readable

    
    $('<option />', {
        'val': result[item],
        'text': result[item]
    })